src/main/java/com/purbon/kafka/topology/backend/FileBackend.java

```diff
@@ -34,7 +34,7 @@
 
   private RandomAccessFile writer;
   private String expression =
-      "^\"?\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\'\"?";
+      "^\"?\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(\\S+)\\',\\s*\\'(.+)\\',\\s*\\'(\\S+)\\'\"?$";
```

Comment:
@sverrehu @akselh instead of using .+ do you we better use something like: _([\w+|[\s|-]?]+)_

this will cover 

```
,foo bar,
,foo123,
,foo-bar,
```
but not the empty name.

```
,,
```

what do you think?

Reply:
That regexp makes the non-trivial tests fail:
```
java.io.IOException: line ('CLUSTER', 'Topic', 'host', 'op', 'User:C=NO,CN=John Doe,emailAddress=john.doe@example.com', 'LITERAL') does not match

java.io.IOException: line ('CLUSTER', 'Topic', 'host', 'op', 'SPIFFE:spiffe://example.com/foo/bar', 'LITERAL') does not match
```

Reply:
I wonder why do we need to support such kind of very long names. While I kinda think the second might make sense, the first one is clearly an anti pattern as it uses the full cert subject name.

Reply:
for now I'll merge it as is, but the regexp is really general, very good to tie them up later I guess.